### PR TITLE
Added likelihood fit to BeamPixel DQM application

### PR DESCRIPTION
**Pull request by dinardo**
I've added a likelihood fit for histograms fitted with a Gaussian.
Before they were fitted with a chi2 method but since there might be empty bins, likelihood is more appropriate.

### CODE REVIEW

**Comment by dmitrijus** [comment]
Your forgot to bump the version field in the master PR.

**Comment by dinardo** [comment]
Hi Dmitrijus,
isn't it this one?
https://github.com/cms-sw/cmssw/pull/23261

What's the "version field" ?

Cheers,
- Mauro.

File: DQM/BeamMonitor/plugins/Vx3DHLTAnalyzer.cc [hunk 1]
```diff
@@ -3,7 +3,7 @@
   \Display Beam-spot monitor entirely based on pixel detector information
            the monitoring is based on a 3D fit to the vertex cloud
   \Author Mauro Dinardo
-  \Version $ Revision: 3.5 $
+  \Version $ Revision: 3.6 $
```
Comment (dmitrijus) [inline DQM/BeamMonitor/plugins/Vx3DHLTAnalyzer.cc]: This is the version field.

**Comment by dmitrijus** [comment]
+1

**Comment by jfernan2** [comment]
backport of #23261
Successfully tested and integrated in the Online DQM at P5. https://twiki.cern.ch/twiki/bin/viewauth/CMS/DQMP5TagCollector
Master PR merged

**Comment by dmitrijus** [comment]
please close this one

**Pull request closed without merging**